### PR TITLE
FeedBuilder: relative paths, roaming, wildcards and 1 bug fix 

### PR DESCRIPTION
**Pull request by lkettenb**
When multiple people working on the same project using FeedBuilder it is neccessary to support relative paths and to activate roaming for all applications settings.

Using wildcards to exclude files from build (replaces checkboxes) is more convenient and makes the config file human readable. Furthermore, one can exclude files from build in config file without using the GUI at all.

When using the config file command line option the applications needs two starts to finally read the new config file. The settings `XmlDocument` needs to be static and set to `null` on reload to fix this bug.

### CODE REVIEW

**Comment by lkettenb** [comment]
Using a string for feeds sometimes results in problems with different encodings (e.g., [byte-order mark](http://www.ipreferjim.com/2014/09/data-at-the-root-level-is-invalid-line-1-position-1/)). But the used workaround does not work on all operating systems (e.g., Windows Server 2012 R2). However, using streams instead seems to do the job.

**Comment by robinwassen** [comment]
This PR is welcome, but quite hard to follow since there seem to be some indentation change that causes the whole files to get changed.

I will review this and then merge it.
